src/test/java/TasksServletTest.java

```diff
@@ -128,124 +205,98 @@ public void setUp() throws IOException, GeneralSecurityException {
     Mockito.when(authenticationVerifier.verifyUserToken(Mockito.anyString()))
         .thenReturn(AUTHENTICATION_VERIFIED);
 
-    request = Mockito.mock(HttpServletRequest.class);
-    response = Mockito.mock(HttpServletResponse.class);
-    Mockito.when(request.getCookies()).thenReturn(validCookies);
-
     // Writer used in get/post requests to capture HTTP response values
     stringWriter = new StringWriter();
     printWriter = new PrintWriter(stringWriter);
+
+    request = Mockito.mock(HttpServletRequest.class);
+    response =
+        Mockito.mock(
+            HttpServletResponse.class,
+            AdditionalAnswers.delegatesTo(new HttpServletResponseFake(stringWriter)));
+
+    Mockito.when(request.getCookies()).thenReturn(validCookies);
     Mockito.when(response.getWriter()).thenReturn(printWriter);
```

Comment:
I don't think you need to mock `response.getWriter()` any more.

Reply:
Yup you're right - it's in `HttpServletResponseFake` already, thanks for catching that!

Reply:
`RESOLVED`